resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Service_MARIONETTECOLLECTIVE">
+      <ComponentRef Id="Service_MARIONETTECOLLECTIVE_svc" />
+    </ComponentGroup>
+
+    <DirectoryRef Id="MARIONETTECOLLECTIVEBINDIR">
+      <Component Id='Service_MARIONETTECOLLECTIVE_svc'
```

Comment:
I guess one way we could do this simply (lets MAINT it though) is to add a serviceref or shortref key in the ```install_service``` method, where we would use this instead of the full ```MARIONCOLLECTIVE``` name. (just thinking on the hoof). If this is not provided, then it would default to the full length name.